### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by jeffdechambeau**
Tweaks & Polish & additional fixes. 
Includes a refactor of the calendaring interface. 
Updates to intention handling.
Bug fixes on barking scenarios.

### CODE REVIEW

File: src/scripts/defaultEvent.js [hunk 1]
```diff
@@ -83,7 +86,7 @@ async function notifyUser(evt) {
             if (type == 'break') break
         }
 
-        break
+        // break
```
Comment (jeznag) [inline src/scripts/defaultEvent.js:89]: Why this change?

Comment (jeffdechambeau) [reply src/scripts/defaultEvent.js:89]: So that the dialog stays present even if the user doesn't respond -- I believe it now times out after 15 min.

Comment (jeznag) [reply src/scripts/defaultEvent.js:89]: can you delete the line rather than commenting it out?
